README.md

```diff
@@ -1,20 +1,42 @@
 # Sample command-line Tool
```

Comment:
needs to be centered as well. also, a small subtitle in italic is missing. please check the template readme again as reference

Reply:
I also think the the title is not how it is supposed to be: "Sample command-line tool"? Maybe "Sample Status CLI", "A command-line tool to...."

Reply:
very good point, is addressed